src/main/java/com/agilethought/internship/sso/services/ServiceApplication.java

```diff
@@ -6,5 +6,6 @@
 public interface ServiceApplication {
 
 	public UserId createUser(User user);
+	public java.util.List<User> getUsers();

```

Comment:
Please add "java.util.List" like a import, the method should be like: "public List<User> getUsers();"